nexus/lib/nexus/peer.ex

```diff
@@ -5,9 +5,45 @@ defmodule Nexus.Peer do
 
   require Logger
 
-  alias ExWebRTC.{ICECandidate, MediaStreamTrack, PeerConnection, RTPCodecParameters}
-  alias Nexus.Chamber
+  alias ExWebRTC.{
+    ICECandidate,
+    MediaStreamTrack,
+    PeerConnection,
+    RTPCodecParameters,
+    SessionDescription
+  }
+
+  alias Nexus.Room
```

Comment:
In the line below, there's alias `alias Nexus.Peer.State` which is probably unnecessary.